pages/awards/[id].js

```diff
@@ -38,6 +38,9 @@ function AwardDetails({ award }) {
   const [confirmationModalOpen, setConfirmationModalOpen] = useState(false);
   const [performanceToFinalize, setPerformanceToFinalize] = useState(-1);
 
+  // Delete award confirmation modal
+  const [deleteConfirmationModal, setDeleteConfirmationModal] = useState(false);
```

Comment:
```suggestion
  const [deleteConfirmationModalOpen, setDeleteConfirmationModalOpen] = useState(false);
```

Could we rename these variables so that it's clearer that they store the state of whether the confirmation model is _open_?